### PR TITLE
:D

### PR DESCRIPTION
**Pull request by pratishrai**
I've tested this code, it works!!!

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,34 +1,23 @@
 import discord
 import asyncio
 import os
-
```
Comment (aadibajpai) [inline bot.py:4]: useful to have this to separate it out

File: bot.py [hunk 2]
```diff
@@ -1,34 +1,23 @@
...
 from datetime import datetime, timedelta
 from discord.ext import commands, tasks
 
 bot = commands.Bot(command_prefix='v;')
 
 
 # target_channel_id = 702296171829264394  # wellness-office in vandy discord
-target_channel_id = 722609125950750771  # testing
+target_channel_id = 722609125950750771 #testing
```
Comment (aadibajpai) [inline bot.py]: actually having a space is better because pep8

File: bot.py [hunk 3]
```diff
@@ -1,34 +1,23 @@
...
-
 @bot.event
 async def on_ready():
     await bot.change_presence(status=discord.Status.online,
-                                 activity=discord.Activity(name="the clock.", type=3))
-    print(f'{bot.user.name} is running now')
-    remaining = time_to_sleep()
-    print(f"Going to sleep for {remaining} seconds.")
-    await asyncio.sleep(remaining)
+                                 activity=discord.Activity(name="the clock!!!", type=3))
```
Comment (aadibajpai) [inline bot.py]:
```suggestion
                                 activity=discord.Activity(name="the clock.", type=3))
```

File: bot.py [hunk 4]
```diff
@@ -44,20 +33,22 @@ async def on_message(message):
...
+@tasks.loop(minutes=30)
 async def daily_purge():
     global activity
     if activity:
-        print("Purge delayed for 30 mins.")
+        print("Purge snoozed for 30 minutes")
     else:
         purge_channel = bot.get_channel(target_channel_id)
         await purge_channel.send(f"Messages about to be purged in `10` seconds in channel {purge_channel.mention}")
-        print('About to yeet.')
```
Comment (aadibajpai) [inline bot.py:55]: I think we can keep this one for logs

Comment (pratishrai) [reply bot.py:55]: yeah

File: bot.py [hunk 5]
```diff
@@ -44,21 +34,26 @@ async def on_message(message):
...
+        print('About to be yeeted.')
         await asyncio.sleep(10)
-        await purge_channel.purge()
-        await purge_channel.send("Yeeted.")
-
-        remaining = time_to_sleep()
+        deleted = await purge_channel.purge()
+        await purge_channel.send(f"Yeeted {deleted} messages.")
```
Comment (pratishrai) [inline bot.py]: `await purge_channel.send(f"Yeeted {len(deleted)} messages.")`

Comment (aadibajpai) [inline bot.py]:
```suggestion
        await purge_channel.send(f"Yeeted {len(deleted)} messages.")
```

**Pull request merged**